### ISSUE TITLE
Fix travis

### ISSUE THREAD

**Issue by DevOrc** [DevOrc/aurora#7]
[Travis](https://travis-ci.com/DevOrc/aurora) is currently failing for some reason:
```
panicked at 'Unexpected token type: Identifier("sub")', src/expr.rs:72:18
```

Lua Source: 
```lua
add = 5 + 5
sub = 5 - 5
mult = 5 * 5
div = 5 / 5

print("5 + 5 = " .. add)
print("5 - 5 = " .. sub)
print("5 * 5 = " .. mult)
print("5 / 5 = " .. div)
```

Rust Source
```rust
fn scan_stmt(&mut self) -> Stmt{
        let token = self.next_token();

        if token == None{
            return Stmt {stmt_type: StmtType::EOF};
        }

        let token = token.unwrap();

        if self.expr_type == ExprType::SingleValue{
            return Stmt{stmt_type: StmtType::Value(token)};
        }

        match token{
            Token::NumberLiteral(_) => self.scan_num_expr(token),
            Token::StringLiteral(_) => self.scan_string_expr(token),
            _ => panic!("Unexpected token type: {:?}", token) //  <- Panics here
        }
    }
```

**Comment by DevOrc** [comment]
It is passing on my computer so I don't why its failing on travis

**Comment by DevOrc** [comment]
fixed by 08c26b1f589d12598de2502338e17b60df423c83
